lighthouse-core/report/html/templates.html

```diff
@@ -891,3 +891,24 @@
     </div>
   </div>
 </template>
+
+<template id="tmpl-lh-3p-filter">
+  <style>
+    .lh-3p-filter {
+      background: var(--lh-table-higlight-bg);
+      color: var(--medium-75-gray);
+      float: right;
+      padding: 6px;
+    }
+    .lh-3p-filter-label, .lh-3p-filter-input {
+      cursor: pointer;
+      vertical-align: middle;
+    }
+  </style>
+  <div class="lh-3p-filter">
+    <label class="lh-3p-filter-label">
+      <input type="checkbox" class="lh-3p-filter-input" checked />
+      Show 3rd party resources (<span class="lh-3p-filter-count"></span>)
```

Comment:
right now this is hardcoded english. Can you replace the `Show 3rd party resources` here with something you add to `Util.UIStrings`?  

You can leave the `(2)` thing mostly as is. We don't support this completely correctly but i'm going to try to fix that now... :)

Reply:
bump on this :) I think we decided we still want to UIString this even if we don't support plurals

Reply:
Ah yes. Thanks. :)